includes/features/SU-SWS/stanford_courses/stanford_courses.feature

```diff
@@ -3,26 +3,26 @@ Feature: Stanford Courses
   As an administrative user
   I want to ensure that the Stanford Courses module is working properly
 
-  @api
+  @api @safe
   Scenario: Verify Stanford Courses is enabled
     Given I am logged in as a user with the "administrator" role
     And I am on "/admin/modules"
     Then the checkbox "Stanford Courses" should be checked
 
-  @content @live
+  @live
   Scenario: View current courses and Featured Course sidebar block
     Given I am on "courses"
     Then I should see "Current Courses"
     And I should see the "Featured Course" heading in the "First sidebar" region
 
-  @content @live
+  @live
```

Comment:
@kmakitan
When the site is live, this placeholder content should be removed. This test should be @deploy. 
However, please make tests that verify this content has been removed for @live.